corporate/tests/test_stripe.py

```diff
@@ -1972,8 +1973,12 @@ def test_upgrade_by_card_with_outdated_seat_count(self, *mocks: Mock) -> None:
         _, context_when_upgrade_page_is_rendered = billing_session.get_initial_upgrade_context(
             initial_upgrade_request
         )
-        # Change the seat count while the user is going through the upgrade flow
+        # Change the seat count in upgrade flow: after do_upgrade, during process_initial_upgrade
         with (
+            patch(
+                "corporate.lib.stripe.BillingSession.stale_seat_count_check",
```

Comment:
Because we now do a current billable license check in `do_upgrade`, we need to patch this here so that there's still a billing inconsistency that's found in `process_initial_upgrade`.